### ISSUE TITLE
Error in decoder.py on start

### ISSUE THREAD

**Issue by Tardog** [DragonComputer/Dragonfire#47]
After installing Dragonfire, Kaldi, festival, and Julius, I get the following error when starting the application from a shell:

```Traceback (most recent call last):
  File "/usr/bin/dragonfire", line 11, in <module>
    load_entry_point('dragonfire', 'console_scripts', 'dragonfire')()
  File "/home/tardog/Dokumente/Spielweise/Dragonfire/dragonfire/__init__.py", line 544, in initiate
    start(args)
  File "/home/tardog/Dokumente/Spielweise/Dragonfire/dragonfire/__init__.py", line 60, in start
    recognizer = KaldiRecognizer()
  File "/home/tardog/Dokumente/Spielweise/Dragonfire/dragonfire/sr/kaldi.py", line 34, in __init__
    self.decoder_pipeline = DecoderPipeline({"decoder" : decoder_conf})
  File "/home/tardog/Dokumente/Spielweise/Dragonfire/dragonfire/sr/decoder.py", line 17, in __init__
    self.create_pipeline(conf)
  File "/home/tardog/Dokumente/Spielweise/Dragonfire/dragonfire/sr/decoder.py", line 47, in create_pipeline
    self.asr.set_property(key, val)
AttributeError: 'NoneType' object has no attribute 'set_property'
```

Despite that error, it appears to run (icon in the status bar), but Julius is not started. Even if I do start Julius manually, Dragonfire doesn’t react to any of my voice commands.

**Comment by mertyildiran** [comment]
@Tardog sorry [this document](https://github.com/DragonComputer/Dragonfire/blob/master/CONTRIBUTING.md) is outdated a little bit. We completely abandoned Julius.

The error is related to Kaldi. Either an environment variable named `GST_PLUGIN_PATH` is not set so set it with `export GST_PLUGIN_PATH=/usr/share/kaldi/src/gst-plugin` before launching Dragonfire. 

OR: Something bad happened on installation and there is no Kaldi(compiled) available under `/usr/share/kaldi` so try to run this script after cloning the repository: https://github.com/DragonComputer/Dragonfire/blob/master/install.sh

**Comment by Tardog** [comment]
@mertyildiran Ah, got caught in the outdated documentation trap. Good to know I don’t need Julius after all.

Setting `GST_PLUGIN_PATH` got rid of the above error – thank you for pointing me in the right direction.

Is the environment variable supposed to be set during installation? If not, it would be helpful to add this piece of information to the troubleshooting section or the readme.

**Comment by mertyildiran** [comment]
@Tardog fixed with a5d0eecf9e38e1bb572b9ba89f652b63ec1b9c02
